corehq/apps/domain/migrations/0019_appmanagerdomainsettings_show_release_mode.py

```diff
@@ -0,0 +1,40 @@
+from django.db import migrations, models
+
+from corehq.apps.domain.models import (
+    AppManagerDomainSettings,
+    AppReleaseModeSetting,
+)
+from corehq.util.django_migrations import skip_on_fresh_install
+
+
+@skip_on_fresh_install
+def migrate_app_release_mode_setting(apps, schema_editor):
+    """
+    Copy ``AppReleaseModeSetting.is_visible`` to
+    ``AppManagerDomainSettings.show_release_mode``.
+    """
+    for setting in AppReleaseModeSetting.objects.filter(is_visible=True):
```

Comment:
There are only 383 matching rows on Production -- This migration should not need to be run separately before deploy.

Reply:
(Sorry for backing out of this commit, and the confusion this comment might cause. It turns out that this change is a luxury I don't have time for.)